.circleci/config.yml

```
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@0.10.0
+  architect: giantswarm/architect@0.17.0
```

```suggestion
  architect: giantswarm/architect@0.18.1
```

Can we use this so we have the latest orb?

---

Updated thanks for pointing that out :)